docker-compose.yaml

```diff
@@ -19,3 +19,26 @@ services:
     command: 'pserve c2c://${INI_FILE}'
     ports:
       - 8080:6543
+
+  minio:
+    image: minio/minio
+    ports:
+      - "9000:9000"
+    command: 'server start'
+
+  # minio-client:
```

Comment:
@arnaud-morvan @tonio  I don't get minio-client to run after minio (server) even though using 'depends_on'. Is it because entrypoint does not wait for the container to be ready? Any idea how to handle this?

It means currently thinkhazard bucket is not created automatically for dev, but must be created via minio webinterface.